static/js/import_wizard2/components/mapping_page.tsx

```diff
@@ -55,9 +56,32 @@ export function MappingPage(props: MappingPageProps): JSX.Element {
   }
 
   useEffect(() => {
-    // TODO(beets): Use server-side detection API.
-    const predictedMapping = getPredictions(props.csvData, placeDetector);
-    setPredictedMapping(predictedMapping);
+    /* Build JSON of column data for detection API Request */
+    const column_ids = new Map(
```

Comment:
we want to use camelcase for variables in typescript.

Reply:
let's also pull this out into a helper function so that we can easily add tests

Reply:
Moved to a helper function!